packages/elements/src/containers/API.stories.tsx

```diff
@@ -42,13 +42,13 @@ APIWithJSONProvidedDirectly.storyName = 'Direct JSON Input (Zoom)';
 
 export const APIWithoutDescription = Template.bind({});
 APIWithoutDescription.args = {
-  apiDescriptionDocument: simpleApiWithoutDescription,
+  apiDescriptionDocument: JSON.stringify(simpleApiWithoutDescription, undefined, 2),
```

Comment:
I don't know why, but I always use `null` for this... but it's the same 🤷‍♂️ 

```suggestion
  apiDescriptionDocument: JSON.stringify(simpleApiWithoutDescription, null, 2),
```

Reply:
I know, team void crew doesn't like `undefined` 😂

Reply:
In the other place in the file there is also `null` used, so I am changing to `null`, for consistency's sake. You win!

Reply:
hehe, no I was actually a silent spy regarding `undefined` and secretly changed `void 0` wherever they wouldnt notice 
🤫  🥷